client/app/vps/additional-disk/order/vps-order-additional-disk.html

```diff
@@ -2,60 +2,49 @@
 
     <div class="row d-lg-flex">
         <div class="col-xm-12">
-
-            <cui-wizard-form data-form-title="{{ 'vps_order_additionnal_disk_title' | translate }}"
-                             data-form-loaded="true"
-                             data-form-on-cancel="$ctrl.cancel()"
-                             data-form-on-complete="$ctrl.confirm()">
-
-                <cui-wizard-step data-step-title="{{ 'common_order' | translate }}"
-                                 data-step-on-init="$ctrl.getAdditionalDiskPrices()"
-                                 data-step-loaded="!$ctrl.loaders.capacity"
-                                 data-step-completed="$ctrl.model.capacity">
-                    <div class="text-center" data-ng-if="$ctrl.loaders.capacity">
-                        <oui-spinner></oui-spinner>
-                    </div>
-                    <div data-ng-if="!$ctrl.loaders.capacity">
-                        <p data-translate="vps_order_additional_disk_select_capacity"></p>
-                        <div class="row-fluid" data-ng-repeat="capacity in $ctrl.capacityArray">
-                            <div class="radio offset1">
-                                <label>
-                                    <input type="radio"
-                                           data-ng-model="$ctrl.model.capacity"
-                                           name="capacity"
-                                           data-ng-value="{{capacity.size}}">
-                                    {{ capacity.type | additionalDiskOptions }} {{ "unit_size_GB" | translate}}
-                                    <span class="red" data-translate="vps_order_additional_disk_select_capacity_price_tax_free"
-                                          data-translate-values="{price: capacity.text}"></span>
-                                </label>
-                            </div>
+            <h1 class="oui-header_1">
+                {{ ::'vps_order_additionnal_disk_title' | translate }}
```

Comment:
You can directly use `data-translate` on `<h1>`

Reply:
This is fixed :)